.gitignore

```diff
@@ -6,3 +6,4 @@ test/spago-test
 **/__pycache__
 templates/docs-search-app.js
 templates/purescript-docs-search
+.DS_Store
```

Comment:
I see the file commited. You should use `git rm` to get rid of it.

Reply:
Ahh yes. Sorry about that, removed now!